classes/02class/exercises/c02-network02/christina-roperto/c02-network02.txt

```diff
@@ -0,0 +1,28 @@
+# Subnetting
+Given a AWS network range, slice it in 4 small subnets and identify how many usable hosts for each one of them.
+
+## 10.0.0.0/16
+/16 = 65,536 addresses 
```

Comment:
not entirely true stating the number of hosts per subnet here as EACH subnet will "remove" 5 usable hosts

maybe just mention that will depend on the number of existing subnets?

Reply:
Hi Caio,

I meant 10.0.0.0/16 has 65,536 addresses (before slicing it into 4 subnets) and 65,531 usable hosts. Once it's sliced into 4 subnets, each subnet will have 16,379 usable hosts. Does that make sense? Let me know if I have to clarify better.